chainregistry.go

```diff
@@ -261,6 +261,7 @@ func newChainControlFromConfig(cfg *config, chanDB *channeldb.DB,
 		walletConfig.ChainSource = chain.NewNeutrinoClient(
 			activeNetParams.Params, neutrinoCS,
 		)
+		time.Sleep(1 * time.Second)
```

Comment:
IIRC we deduced the precise reason why we need this?

Reply:
Based on the logs @guggero sent to me, I was able to find https://github.com/lightninglabs/neutrino/pull/200. This solved a flake for him, but I believe there's another one we're still investigating.

Reply:
The neutrino fix definitely removed the need for the `time.Sleep`.
There's still a flake in the SCB restore:
```
        --- FAIL: TestLightningNetworkDaemon/channel_backup_restore/restore_during_unlock (41.49s)
            lnd_test.go:106: Error outside of test: *errors.errorString unable to find Carol's force close tx in mempool: wanted 1, found 0 txs in mempool: []
                /home/guggero/projects/go/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_test.go:8555 (0xdc9db3)
                        assertDLPExecuted: t.Fatalf("unable to find Carol's force close tx in mempool: %v",
                /home/guggero/projects/go/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_test.go:14116 (0xdec0aa)
                        testChanRestoreScenario: assertDLPExecuted(
                /home/guggero/projects/go/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_test.go:14437 (0xe08532)
                        testChannelBackupRestore.func7: testChanRestoreScenario(h, net, &testCase, password)
                /usr/lib/go-1.13.1/src/testing/testing.go:909 (0x50ac19)
                        tRunner: fn(t)
                /usr/lib/go-1.13.1/src/runtime/asm_amd64.s:1357 (0x460e81)
                        goexit: BYTE    $0x90   // NOP

```
It does also happen with `btcd` although much less often than with neutrino.
Should we continue with this PR without a fix for it and move that to a follow-up PR?